test/remote/gateways/remote_worldpay_test.rb

```diff
@@ -43,6 +43,7 @@ def test_authorize_and_capture
     assert_success auth
     assert_equal 'SUCCESS', auth.message
     assert auth.authorization
+    sleep(40)
```

Comment:
Doh.. I suppose something changed on their sandbox that requires the sleep statement now.

Reply:
Yea but unfortunately it doesn't work when all the test run, but if I run the test one by one they pass.